### PR TITLE
Logger utc

### PR DESCRIPTION
**Pull request by Cashmaney**

### CODE REVIEW

File: src/common/logger.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 const log4js = require("log4js");
 const constants = require("./constants");
 const LOG_CONFIG = constants.LOG_CONFIG;
+const format = require("date-format");
 
 class Logger {
   constructor(options = {}) {
-    const logName = options.hasOwnProperty("name") ? options.name : "P2P";
+    const logName = options.hasOwnProperty("name") ? options.name : "Logger";
```
Comment (lenak25) [inline src/common/logger.js:8]: Why did you change the logger name?

**Review by lenak25 — DISMISSED** [review]
LGTM

**Comment by assafmo** [comment]
@Cashmaney It seems npm cannot install `date-format` v3.0.3 because v3.0.0 is the latest version
https://www.npmjs.com/package/date-format
```bash
$ npm install
npm ERR! code ETARGET
npm ERR! notarget No matching version found for date-format@^3.0.3
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'enigma-p2p'
npm ERR! notarget 
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-12-10T21_59_47_805Z-debug.log
The command "npm install" failed and exited with 1 during .
```

**Comment by assafmo** [comment]
It now fails from the same reason as #270 & #271 & develop

**Review by lenak25 — APPROVED** [review]
